client/src/pages/CreateProfilePage/CreateProfilePage.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import './CreateProfilePage.css';
+import '../ProfilePage/ProfilePage.css';
+import Tag from '../../components/Tag/Tag';
+import instagram from '../../assets/instagram.svg';
+import facebook from '../../assets/facebook.svg';
+import tina from '../../assets/tina.png';
+import plusSign from '../../assets/plus-sign.svg';
+import logo from '../../assets/CoFEDlogo.png';
+import { Link } from 'react-router-dom';
+import '../../components/Navbar/Navbar.css';
+
+export default function Profile(props) {
```

Comment:
as bianca said above, this file seems redundant and can be solved by putting something like an `editOnly` prop in `ProfilePage` that when true, unconditionally returns `renderEdit()`. in general, whenever you find yourself copy pasting a large amount of code, there's probably a much better way to solve the issue